magenta_docker/config_multiple_vms.yaml

```diff
@@ -0,0 +1,21 @@
+# Recommended batch_size: 128
+# Recommended learning_rate: 0.001
+# Recommended number of steps: 15000
```

Comment:
You can add in the comment the link to the design doc explaining how you got this results.

Reply:
Even for the PR to the Magenta library? It won't be externally visible

Reply:
I will add the Markdown files inside our repo when the design doc is in final stage.